docs/superpowers/plans/2026-04-11-spend-service.md

```
@@ -0,0 +1,793 @@
+}
+```
+
+- [ ] **Step 2：執行測試，確認全部通過**
+
+```bash
+cd backend && go test ./internal/services/ -run "TestRedeem" -v
+```
+
+期望輸出：
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)